### DISCUSSION TITLE
how to scroll preview

### DISCUSSION THREAD

**Discussion by fastfading** [sxyazi/yazi#466] [1. Q&A] [answered]
what is the shortcut key for scroll preview
there is no document for that

**Comment by sxyazi** [comment]
Use the `peek` command:

https://github.com/sxyazi/yazi/blob/283f9377f27202c9dc4496774b6b7866ca4b7677/config/preset/keymap.toml#L28-L29

The docs here: https://yazi-rs.github.io/docs/configuration/keymap#navigation

**Reply by fastfading** [reply]
this works on pc keyboard layout 
how about mac , no alt key on mac,   alternative "option" key does not work

**Reply by sxyazi** [reply, marked as answer]
This depends on the terminal emulator you're using.

Such as kitty, you need to enable [macos_option_as_alt](https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.macos_option_as_alt) to treat `Option` as `Alt` because `Option` is specific to macOS, and the terminal protocol doesn't support the `Option` key.

From what I know, most macOS terminals also consider Option as Alt, but kitty is an exception that requires you to set it separately.

**Reply by QYQSDTC** [reply]
I am using iTerm2, I didn't find using `Option` as `Alt` in settings, my settings are below, the `Option` doesn't work so I changed it to `CTRL` in the `keymap.tol` and it worked, but only work once, i.e., if I am previewing a PDF with many pages, after I press `CTRL+j` it will jump to next page, but can only jump once, then this key binds don't work anymore, no matter i press `CTRL+k` or `CTRL+j` it just stay still.
![CleanShot-6ksM4XJF@2x](https://github.com/sxyazi/yazi/assets/15608306/d79114ef-8517-4512-8caa-d040971491b6)

**Reply by sxyazi** [reply]
what‘s your Yazi version?

**Reply by QYQSDTC** [reply]
![CleanShot-MYRp5DEQ@2x](https://github.com/sxyazi/yazi/assets/15608306/93b0bce9-3e78-40a9-8d99-d76d4df484dd)

**Reply by sxyazi** [reply]
It sounds like a bug that's not relevant to this discussion, would you like to file an issue for that?

**Comment by fastfading** [comment]
you are right         it works on my mac now

**Comment by xxyxxyxyx1** [comment]
@sxyazi, please consider adding more about this feature to the documentation. I searched for a long time how to activate preview scrolling, and ended up on this page only after a lot of frustration and luck. 

The Features page demonstrates "Scrollable Preview", so I searched "scroll" and "scrollable" in the documentation. These did not lead to anything. The description of "seek" is not discoverable.

**Reply by sxyazi** [reply]
Thanks for the feedback, done in https://github.com/yazi-rs/yazi-rs.github.io/commit/d305a96020770786c863bcf684cec7d898ec96e2
